### DISCUSSION TITLE
Illegal string offset 'code' 'message' 'body'

### DISCUSSION THREAD

**Discussion by c3aus3scu** [AshleyRedman/WPClothes2Order#107] [Q&A] [answered]
After the last update, I receive this on top of the website.

Warning: Illegal string offset 'code' in /home/daemons/public_html/wp-content/plugins/wpclothes2order/includes/wpc2o-options-orders.php on line 96

Warning: Illegal string offset 'message' in /home/daemons/public_html/wp-content/plugins/wpclothes2order/includes/wpc2o-options-orders.php on line 97

Warning: Illegal string offset 'body' in /home/daemons/public_html/wp-content/plugins/wpclothes2order/includes/wpc2o-options-orders.php on line 98

https://daemons.ro/

**Comment by AshleyRedman** [comment]
@c3aus3scu Thanks for letting me know. This is related to the latest release which shows the response from C2O, however pas orders will not show which is causing the issue. Tomorrow I will release a bug fix

**Comment by AshleyRedman** [comment, marked as answer]
Hi @c3aus3scu I have fixed this bug & release `1.0.8`. Please update your plugin to the latest version
